packages/client/hmi-client/src/utils/log-buffer.ts

```diff
@@ -0,0 +1,83 @@
+// import API from '@/api/api';
+import { useToastService } from '@/services/toast';
+import axios from 'axios';
+import useAuthStore from '../stores/auth';
+
+const LOGS = axios.create({
+	baseURL: '/api'
+});
+
+// Hook in bearer tokens
+// See
+// - https://medium.com/swlh/handling-access-and-refresh-tokens-using-axios-interceptors-3970b601a5da
+LOGS.interceptors.request.use(
+	(config) => {
+		const auth = useAuthStore();
+
+		if (config.headers) {
+			config.headers.Authorization = `Bearer ${auth.token}`;
+		} else {
+			config.headers = { Authorization: `Bearer ${auth.token}` };
+		}
+		return config;
+	},
+	(error) => {
+		console.warn(error);
+	}
+);
```

Comment:
We already have an Axios call API for that same purpose, and the same kind of intereceptors on the App.vue.
I think all of those needs to be refactored.

Reply:
Yeah.  I wasn't exactly sure of what the best approach was.  The one on `app.vue` re routes you to the login page.  The `api.vue` might not have access to this vue router yet when api is being used.  I'm all ears if there's a particular pattern that solves this.